### PR TITLE
msvc build and nuget push to local nuget repo

### PR DESCRIPTION
**Pull request by mary-dcouto**
msvc build file, that build Ziti.nuget.sln and push to local nuget repo

### CODE REVIEW

File: Ziti.NET.Standard/Ziti.NET.Standard.csproj [hunk 1]
```diff
@@ -65,4 +65,8 @@
     <PropertyGroup Condition="'$(Platform)' == 'x64'">
         <DefineConstants>TRACE;ZITI_X64</DefineConstants>
     </PropertyGroup>
+	
+	<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
```
Comment (dovholuknf) [inline Ziti.NET.Standard/Ziti.NET.Standard.csproj:69]: did you add this on purpose? was it necessary for some reason?

File: Ziti.NET.Standard/msvc_build.bat [hunk 2]
```diff
@@ -0,0 +1,102 @@
+@echo off
```
Comment (dovholuknf) [inline Ziti.NET.Standard/msvc_build.bat]: this file is probably misnamed. doesn't seem like any 'msvc' stuff in here. perhaps just call it 'build.bat'?

File: Ziti.NET.Standard/msvc_build.bat [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    echo.
+    echo nuget pack for %BUILD_VERSION% failed
+    echo.
+    goto FAIL
+) else (
+    echo.
+    echo result of dotnet pack: %ACTUAL_ERR%
+)
+
+FOR /F "delims= " %%i IN ('ls -rt Ziti.NET.Standard.*.nupkg ^| tail -n 1') DO set NUPKG_FILE=%%i
```
Comment (dovholuknf) [inline Ziti.NET.Standard/msvc_build.bat]: can you add a comment that this expects 'tail' to be on your path to work? just a comment in the bat file is good enough for now

**Review by dovholuknf — APPROVED** [review]

**Comment by dovholuknf** [comment]
address the comments and then merge away :)

File: Ziti.NET.Standard/Ziti.NET.Standard.csproj (hunk 1 above)
Comment (mary-dcouto) [reply Ziti.NET.Standard/Ziti.NET.Standard.csproj:69]: I got some errors when I tried to run Testproject with AnyCPU platform. So, I added the x86 platform settings in the library and tested the sample code using x86 platform

**Pull request merged**